docs/releases/upgrading.mdx

```diff
@@ -136,6 +136,41 @@ Storybook upgrades are not the only thing to consider: changes in the ecosystem
 
 It runs a set of standard configuration checks, explains what is potentially out-of-date, and offers to fix it for you automatically. It also points to the relevant documentation so you can learn more. It runs automatically as part of [`storybook upgrade`](#upgrade-script) command, but it's also available on its own if you don't want to upgrade Storybook.
 
+<IfRenderer renderer="react">
+  
+### Next.js to Next.js Vite automigration
+
+The `nextjs-to-nextjs-vite` automigration helps you migrate from the Webpack-based `@storybook/nextjs` framework to the Vite-based `@storybook/nextjs-vite` framework, which we recommend for most Next.js projects.
+
+**When it runs:**
+
+This automigration will be detected and offered when you run `storybook automigrate` or `storybook upgrade` if:
+- Your project has `@storybook/nextjs` installed
+- You want to migrate to the faster, more modern Vite-based framework
```

Comment:
This isn't a detection criteria. It's instead a reason to run the migration manually.